wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/SwerveControllerCommandTest.java

```diff
@@ -40,8 +40,8 @@
   private final ProfiledPIDController m_rotController = new ProfiledPIDController(1, 0, 0,
       new TrapezoidProfile.Constraints(3 * Math.PI, Math.PI));
 
-  private static final double kxTolerance = 1 / 12.0;
-  private static final double kyTolerance = 1 / 12.0;
```

Comment:
Could these be made a separate PR and merged first? It'll fix the flaky Azure CI builds too.

Reply:
Sure. I was just playing around to make it pass. Do you think the range I picked is appropriate?

Reply:
I don't remember what the typical deviation is. I'd go with whatever that is with a factor of safety of 2.

Reply:
Ok

Reply:
Now that https://github.com/wpilibsuite/allwpilib/pull/2604/ is merged, you should try the build without the tolerance increase.